### ISSUE TITLE
[Suggestion] Filters for playlist

### ISSUE THREAD

**Issue by shengdie** [soimort/you-get#1288]
Please make sure these boxes are checked before submitting your issue – thank you!

- [x] You can actually watch the video in your browser or mobile application, but not download them with `you-get`.
- [x] Your `you-get` is up-to-date.
- [x] I have read <https://github.com/soimort/you-get/wiki/FAQ> and tried to do so.
- [x] The issue is not yet reported on <https://github.com/soimort/you-get/issues> or  <https://github.com/soimort/you-get/wiki/Known-Bugs>. If so, please add your comments under the existing issue.
- [x] The issue (or question) is really about `you-get`, not about some other code or project.

> I have a suggestion, could you support filters for downloading from a playlist. For example, I want to download a TV-show containing many episodes, but I only want episodes 10-15, however I cannot do this with you-get, because there is no filter option, I have to download all episodes of this TV-show. So hope you could consider my suggestion.

**Comment by cnbeining** [comment]
Yeah...make sense.

**Comment by soimort** [comment]
Personally I prefer this to be accomplished interactively, via curses or even GUI, 'cause it's hard to know which episode(s) you want without seeing their titles.

**Comment by cnbeining** [comment]
How about using ```-i``` to get their information first then download? So we do not break the standard way of how *nix programme works?

**Comment by shengdie** [comment]
Yes, we can get the url of every episode with "-i", but we cannot download
selected episodes by you-get itself, we can use shell "for loop" to deal
with it, it is still very inconvenient.

2016-07-19 22:24 GMT-04:00 David Zhuang <notifications@github.com>:

**Comment by shengdie** [comment]
We can see the titles on the web and decide which to download, or we can
list it with "-i" first.

2016-07-19 21:01 GMT-04:00 Mort Yao <notifications@github.com>:

**Comment by soimort** [comment]
My plan: make a GUI.

Listing playlist contents via `-i`, looking for what you want and then downloading them from the command-line seem like a pain in the ass.

(Note: `youtube-dl` has a similar feature, but, still, you need to find out the "indices" of videos first.)

**Comment by shengdie** [comment]
I've tried youtube-dl, but it is very buggy, many Chinese websites will
split one video as several parts, youtube-dl will take these parts as a
sublist, so the playlist variable will affect it. I wanna say I would have
already known what episodes I want when I need the filters, so nevermind.
Or you might want to add a new option like list the titles only.

2016年7月20日 上午12:28，"Mort Yao" <notifications@github.com>写道：
